docs/plugins/purchase.md

```diff
@@ -88,72 +84,32 @@ Purchase.setup('http://csp-backend.com/api', {
 
-The callback function will receive 2 arguments: `params` and `data` and should _always_ return a `Promise` which resolves the result (or rejects an error in case of a failure). When a callback is specified, all other keys in that `endpoint` config
-are ignored.
+### assets
 
+The `assets` method returns a *Promise* with the result of the request that is made to the CSP to retrieve available assets for purchase. Typically, it returns an Array of objects containing information about the assets (such as *title*, *description* and *price*).
 
-```js
-Purchase.setup('http://csp-backend.com/api', {
-  asset: {
-    callback(params, data) {
-      return Promise((resolve, reject) => {
-        Api.getAssetDetails(params.id)
-          .then(resolve)
-          .catch(reject)
-      }
-    }
-  },
-  signature: {
-    callback(params, data) {
-      return fetch('http://csp-backend.com/api/' + data.household + '?productId=' + params.id, {
-        headers: {
-          'Accept': 'application/json',
-				  'Content-Type': 'application/json'
-        }),
-        method: 'POST',
-      })
-    }
-  },
-})
-```
-
-### Assets
-
-The `assets` method returns a Promise with the result of the request to the CSP to retrieve assets available for purchase. In most cases it would return an `Array` of objects with information about the assets (i.e. _title_, _description_ and _price_).
-
-The `household` is automatically sent as a `query param` in the request, allowing to optionally return information on whether the asset has been _purchased_ by the current household.
-
-The `assets` method is an _optional_ convenience method exposed by the Purchase plugin. Depending on the CSPs API integration, the functionality to retrieve assets can be implemented fully custom in an App.
+You can use the `household` option to return information on whether the assets were purchased by the current household or not. It is automatically added as a `query param` in the request.
 
+The `assets` method is an *optional* convenience method provided by the Purchase plugin. Depending on the CSP's API integration, you can implement the functionality to retrieve assets in your App in a fully customized manner.
 
 ```js
 Purchase.assets().then(assets => {
   // do something with assets
 })
 ```
 
-### Asset
+### asset
 
 The `asset` method returns a Promise with the result of the request to the CSP to retrieve details for a specific asset.
 
-The `household` is automatically sent as a `query param` in the request, allowing to return information on whether the asset has been _purchased_ by the current household.
+You can use the  `household` option to return information on whether the asset was purchased by the current household or not. It is automatically added as a `query param` in the request.
```

Comment:
same comment as before. I think the new sentence isn't more clear than the old one

Reply:
See above.